data/json/items/classes/comestible.json

```diff
@@ -0,0 +1,61 @@
+[
+  {
+    "id": "food",
+    "type": "COMESTIBLE",
+    "name": "edible comestible",
+    "comestible_type": "FOOD",
+    "volume": 1,
+    "symbol": "%"
+  },
+  {
+    "id": "fruit",
+    "copy-from": "food",
+    "type": "COMESTIBLE",
+    "name": "base fruit",
+    "material": "fruit",
+    "color": "red",
+    "vitamins": [ [ "vitC", 15 ] ]
+  },
+  {
+    "id": "fruit_berry",
+    "copy-from": "fruit",
+    "type": "COMESTIBLE",
+    "name": "fruit berries",
+    "stack_size": 8,
```

Comment:
Same as volume above, the stack size is not a property of berries, in fact they never had it before, why did you add it?